interview-questions.md

```diff
@@ -10,58 +10,81 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is `this` in JavaScript?
 
   Your answer:
+Refers back to the object.
 
   Researched answer:
-
+The JavaScript "this" keyword refers to the object it belongs to.
+It has different values depending on where it is used:
 
 
 2. What is React? Why would you use it?
 
   Your answer:
+  A web framework for JavaScript.
 
   Researched answer:
-
+React is an open-source JavaScript library that is used for building user interfaces specifically for single-page applications.
 
 
 3. Which lifecycle method is required in a React class component?
 
   Your answer:
+Render
 
   Researched answer:
-
+render() method is the only required method in a class component.
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
   Your answer:
 
-  Researched answer:
+JSX is a JavaScript language. You must return a single parent of JSX will not compile.
 
+  Researched answer:
 
+It's an extension of JavaScript that allows developers to write HTML right within JavaScript.
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
   Your answer:
 
+package mangager.
+
   Researched answer:
+package manager for the JavaScript programming language.
```

Comment:
Yes, yarn will connect dependencies. The yarn.lock, package.json, and node_nodules will be updated.